controllers/routemonitor/routemonitor_controller.go

```diff
@@ -78,25 +87,29 @@ func (r *RouteMonitorReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error
 		return ctrl.Result{Requeue: false}, nil
 	}
 
+	log.Info("Entering CreateBlackBoxExporterResources")
 	// Should happen once but cannot input in main.go
 	err = r.CreateBlackBoxExporterResources(ctx)
 	if err != nil {
 		return ctrl.Result{Requeue: true}, err
 	}
 
+	log.Info("Entering GetRoute")
 	route, err := r.GetRoute(ctx, routeMonitor)
 	if err != nil {
 		return ctrl.Result{Requeue: true}, err
 	}
 
-	res, err := r.UpdateRouteURL(ctx, route, routeMonitor)
+	log.Info("Entering UpdateRouteURL")
```

Comment:
not sure we have to constantly log that on info level

Reply:
same as before (overhaul)